City-of-Helsinki/drupal-employment-services
# THF-511: Drupal module and development tool updates

**To test**
- Set up your local environment on the `develop` branch
- Checkout this branch and run `composer install;drush updb -y;drush cim -y`
- Check that everything works
- Set up your local environment again on this branch:
  - `make down up`
  - `make drush-sync-db drush-updb drush-cim drush-cr drush-uli` 
  - All of the above local development commands should work
- Test that the Drupal admin UI and Elasticsearch index works properly
- Read code changes
